catalog/task/get-ocp-version/0.1/get-ocp-version.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-ocp-version
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to collect OCP version tag from FBC fragment using `skopeo inspect`
+  params:
+    - name: fbcImage
+      description: An FBC image to inspect
+      type: string
+  results:
+    - name: store-version
+      description: Store OCP version number from given Image
+  steps:
+    - name: get-ocp-version
+      image: quay.io/hacbs-release/release-utils:4d8649dbb2b626f5fe9f4ff83c1bc3be268fad31
+      script: |
+       #!/usr/bin/env sh
+       set -eux
+
+      # The value to be checked
+      value=$(skopeo inspect --raw docker://"$(params.fbcImage)" | jq '.annotations."org.opencontainers.image.base.name"' |  perl -pe 'if(($v)=/(v[0-9]+([.][0-9]+)+)/){print"$v\n";exit}$_=""')
+
+      # Define the regular expression
+      pattern="^v[0-9]\.[0-9]+$"
+
+      # Check if the value matches the pattern
+      if echo "$value" | grep -Eq "$pattern"; then
+          echo "Valid format."
+      else
+          echo "Invalid format or value does not exist or does not match the required pattern."
+      fi
+ 
```

Comment:
See https://github.com/redhat-appstudio/release-service-bundles/blob/main/catalog/task/collect-data/0.1/collect-data.yaml#L80 for an example of storing a task result. You will probably do like `echo $value | tee result` or something